Update career progression content (#4395)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md

---
title: "Career progression in teaching"
heading: "Career progression in teaching"
description: |-

>   Find out about all the opportunities for progression as a teacher, including early career support, salary progression, and professional qualifications.

Author: ```suggestion
  Find out about the opportunities for progression as a teacher, including salary progression, professional qualifications, and mentorship.
```

## Salary progression

Qualified teachers get a minimum starting salary of $salaries_qualifiedteacher_englandmin$ (or more in London). Your salary will be reviewed every year, with most teachers moving up the pay range annually. This will depend on your school’s performance management arrangements. 

A typical teacher could earn at least $salaries_fiveyears_max$ (or more in London) after 5 years.  

[Find out more about teacher pay](/life-as-a-teacher/pay-and-benefits/teacher-pay). 

### Additional payments

You may also receive additional payments on top of your regular salary for taking on extra responsibilities. These are known as teaching and learning responsibility (TLR) payments. 

This could be a one-off payment for a specific project, for example, creating a timetable. Or it might be an extra responsibility attached to your job role, for example, being the head of department. 

Depending on your responsibilities, you could earn up to an extra $salaries_tlr_max$. Your school will determine how much you receive.

> ## National professional qualifications

Reviewer: Should NPQs come before salary? Feels like they are linked to the section before?

National professional qualifications (NPQs) are professional development courses for teachers and leaders to develop skills, progress careers and improve school outcomes.  

> There are 2 types of NPQs: specialist and leadership. Specialist NPQs can help you develop your classroom teaching or specialism. Leadership NPQs can help you develop your leadership skills or to progress to a leadership role.

Reviewer: Not sure if it is worth mentioning funding - there is a small para at the end of [the GOV.UK page](https://www.gov.uk/guidance/national-professional-qualification-npq-courses) on this (but that might be too much detail)

Would be good to review this with the CPD team

There are lots of opportunities for progression and self development. For example, you could: 

* gain professional qualifications  
* take on additional responsibilities, like pastoral care roles 
* advance into leadership roles 

$aaron$

## Early career progression

The thought of teaching your own class after you’ve completed your teacher training may feel a bit daunting. But teaching offers the opportunity for continuous professional development so you’re always learning new skills.  

If your course leads to qualified teacher status (QTS), you’ll receive 2 years of further support once you’ve completed your training. You’ll be known as an early career teacher (ECT) during this time and receive training and support based on the early career framework (ECF).

[Find out more about early career teachers](/life-as-a-teacher/teaching-as-a-career/early-career-teachers).

## Taking on more responsibility

> Whether it’s during your time as an early career teacher, or at other stages in your career, you can take on more responsibility to help progress your career.

Author: ```suggestion
Whether it’s during your time as an early career teacher, or at other stages in your career, you can take on more responsibility to help you progress and develop.
```

### Pastoral care roles

Pastoral care in schools is about looking after the emotional and physical wellbeing of pupils. Pastoral care roles such as form tutors or [special educational needs co-ordinator (SENCO)](/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs), are great ways to progress your career and develop new skills. 

They give you the opportunity to be the link between all aspects of a pupil's education. You’ll help bring together classroom teachers, senior leadership, parents, carers, extended family and the wider community, to ensure pupils get the support they need.

### Leadership roles

Leadership roles in school can include: 

* subject leader 
* head of department 
* head of year or key stage 
* deputy headteacher 
* headteacher 

These roles tend to combine elements of pastoral care, academic and curriculum management, and staff development.  

$gloria$

> ### Become a mentor

Author: ```suggestion
### Mentor roles
```
